python/sglang/srt/managers/tokenizer_manager.py

```diff
@@ -195,9 +195,13 @@ async def _tokenize_one_request(
         obj: Union[GenerateReqInput, EmbeddingReqInput],
     ):
         """Tokenize one request."""
+        input_embeds = None
         # Tokenize
         input_text = obj.text
-        if obj.input_ids is None:
+        if obj.input_embeds is not None:
+            input_embeds = obj.input_embeds
+            input_ids = obj.input_ids
```

Comment:
we can assert `self.server_args.disable_radix_cache = False` here and return and raise a useful warning message.